mungegithub/mungers/submit-queue.go

```diff
@@ -1060,9 +1070,14 @@ func (sq *SubmitQueue) validForMergeExt(obj *github.MungeObject, checkStatus boo
 
 	// Must pass CLA checks
 	if gateCLA {
-		if !obj.HasLabel(claHumanLabel) && !obj.HasLabel(cncfClaYesLabel) {
-			sq.SetMergeStatus(obj, noCLA)
-			return false
+		for i, l := range sq.ClaYesLabels {
```

Comment:
It is not safe to access this variable from this function without the options lock. Save a reference to the slice while the options lock is held above (where `gateCLA` is set).